{
  try {
    test1();
    test2();
    test3();
    System.out.println("PASSED");
    System.exit(0);
  }
 catch (  Exception e) {
    System.out.println(e);
    System.exit(1);
  }
}
